src/selectorForSlice.js

```diff
@@ -44,3 +42,11 @@ export default function selectorForSlice (slicePath) {
     }
   }
 }
+
+/**
+ * @ignore
+ * @callback SliceSelector
```

Comment:
I'm not sure I understand the difference between using a `@callback` tag vs a `@typedef`. Are both valid options for `SliceSelector`?

Reply:
Typdef doesn't work for functions 😭 .

Refer to https://github.com/LaunchPadLab/client-template/pull/436 for more details. It's a little over my head 😆 😅

Reply:
Right. I just re-read the [docs](https://www.loom.com/share/e2abcc1db31c4ee398c963e37bdbd7dc).
I thought they were using `@typedef` to type functions, but it was actually the function params being typed with `@typedef`. Functions should use `@callback`.